### DISCUSSION TITLE
How to link a CTkLabel to a new GUI

### DISCUSSION THREAD

**Discussion by Yug-Desai** [TomSchimansky/CustomTkinter#856] [Q&A] [answered]
**Hey I am quite new to python.**
I have created a simple login form however I would like to use a label at the bottom of the GUI (which has the text "Create an account.") 

I want this label to link/ display a new GUI (top level) but I don't know how. Is there a way I can use a command to give the label so that when I click it, it opens a new GUI.

I am aware that it can be done with a button, but a label looks better.

This is the GUI:
![image](https://user-images.githubusercontent.com/107952614/207547027-2e4e0348-1fbd-4e21-b49a-e87746870ce1.png)

Is there any command with CTkLabel that will allow me to display a top level?
If you have any answers/recommendations on this problem I would appreciate it!

**Comment by Yug-Desai** [comment]
Hey, thank you, however this isnt my problem, i understand how to use top
level for buttons (by having that as a command on the button, but there is
no commands for a label. I want to click on a label which opens a top level
not a button)

**Reply by DarkKnight263** [reply]
yeah sorry, I realised ur problem statement was a bit different ,hence I deleted my reply(idk how its still visible to u).....also i doubt what u r looking for is actually possible

**Comment by Akascape** [comment, marked as answer]
@Yug-Desai Simply bind the CTklabel with the mouse button.
```python
import tkinter
import customtkinter

def new_window():
    customtkinter.CTkToplevel(app)
    
app = customtkinter.CTk()
                
label = customtkinter.CTkLabel(app, text="create a new account", font=("",13))
label.grid(padx=50, pady=50)

label.bind("<Button-1>", lambda event: new_window())
label.bind("<Enter>", lambda event: label.configure(font=("",13,"underline"), cursor="hand2"))
label.bind("<Leave>", lambda event: label.configure(font=("",13), cursor="arrow"))

app.mainloop()               
```

**Comment by Yug-Desai** [comment]
Also im curious about ur code:
When u said, label.bind("<Button-1>", ....  What does the button 1 refer
to?

**Reply by abhaymakes** [reply]
> Also im curious about ur code: When u said, label.bind("<Button-1>", .... What does the button 1 refer to?
> […](#)
> On Thu, 15 Dec 2022, 9:50 PM fmd, ***@***.***> wrote: Legend thank you On Wed, 14 Dec 2022, 6:11 PM Akash Bora, ***@***.***> wrote: > @Yug-Desai <https://github.com/Yug-Desai> Simply bind the CTklabel with > the mouse button. > > import tkinterimport customtkinter > def new_window(): > customtkinter.CTkToplevel(app) > app = customtkinter.CTk() > label = customtkinter.CTkLabel(app, text="create a new account", font=("",13))label.grid(padx=50, pady=50) > label.bind("<Button-1>", lambda event: new_window())label.bind("<Enter>", lambda event: label.configure(font=("",13,"underline"), cursor="hand2"))label.bind("<Leave>", lambda event: label.configure(font=("",13), cursor="arrow")) > app.mainloop() > > — > Reply to this email directly, view it on GitHub > <[#856 (comment)](https://github.com/TomSchimansky/CustomTkinter/discussions/856#discussioncomment-4401835)>, > or unsubscribe > <https://github.com/notifications/unsubscribe-auth/AZXTTZV2JC2RNW7Q2NFKL7DWNHWULANCNFSM6AAAAAAS6FYP7M> > . > You are receiving this because you were mentioned.Message ID: > ***@***.*** > .com> >

`<Button-1>` refers to the left-click of your mouse.

**Comment by Yug-Desai** [comment]
Legend thank you

**Reply by Akascape** [reply]
@Yug-Desai Please mark this qna as answered.
